libs/ngx-charts-on-fhir/src/lib/data-layer/data-layer-manager.service.ts

```diff
@@ -88,7 +88,7 @@ export class DataLayerManagerService {
     private tagsService: FhirChartTagsService,
     private mergeService: DataLayerMergeService
   ) {}
-
+  dataRetrievalError = false;
```

Comment:
I don't know if this will work with on-push components. We may need to use a subject so we can notify the component when it changes.